pages/news-and-events/community-spotlight/index.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="help-page">
+    <breadcrumb :breadcrumb="breadcrumb" title="Community Spotlight" />
+    <page-hero>
+      <h1>Community Spotlight</h1>
+      <br />
+      <p>
+        Hear about how the SPARC community are using the SPARC portal, including
+        Success Stories and Fireside Chats. Also, keep up to date with Community Announcements.
+      </p>
+    </page-hero>
+    <div class="page-wrap container">
+      <community-spotlight-listings :stories="shownStories" />
+    </div>
+    <pagination
+      v-if="allStories.length > pageSize"
```

Comment:
Typo?

Reply:
for `pagination`? That's how the component is spelled in sparc-app.

https://github.com/nih-sparc/sparc-app/blob/master/components/Pagination/Pagination.vue